bundles/org.openhab.binding.kostalinverter/README.md

```diff
@@ -59,6 +65,77 @@ None
+| aktHomeConsumptionGrid    | Number:Power              | Current consumption from grid  |    R    |
+| phaseSelHomeConsumpL1     | Number:Power              | Current consumption, L1  |    R    |
+| phaseSelHomeConsumpL2     | Number:Power              | Current consumption, L2  |    R    |
+| phaseSelHomeConsumpL3     | Number:Power              | Current consumption, L3  |    R    |
+| gridFreq                  | Number:Frequency          | Current frequency on grid  |    R    |
+| gridCosPhi                | Number:Angle              | Current power factor on grid  |    R    |
+| homeConsumptionDay        | Number:Energy             | Total home consumption today  |    R    |
+| ownConsumptionDay         | Number:Energy             | Total own consumption today  |    R    |
+| ownConsRateDay            | Number:Dimensionless      | Total own consumption rate today  |    R    |
+| autonomyDegreeDay         | Number:Dimensionless      | Total autonomy degree day  |    R    |
+| homeConsumptionTotal      | Number:Energy             | Total home consumption  |    R    |
+| ownConsumptionTotal       | Number:Energy             | Total own consumption  |    R    |
+| totalOperatingTime        | Number:Time               | Total operating time  |    R    |
+| current                   | Number:ElectricCurrent    | Current  |    R    |
+| currentDir                | Number:ElectricCurrent    | Current dir  |    R    |
+| chargeCycles              | Number:One                | Total number of charge cycles  |    R    |       
+| batteryTemperature        | Number:Temperature        | Current battery temperature  |    R    |
+| loginterval               | Number:Time               | Set value for loginterval  |    R    |
+| s0InPulseCnt              | String                    | S0-pulse counter  |    R    | 
+| ownConsRateTotal          | Number:Dimensionless      | Total own comsumption rate  |    R    |
+| autonomyDegreeTotal       | Number:Dimensionless      | Total autonomy degree  |    R    |
+| batteryVoltage            | Number:ElectricPotential  | Current battery voltage  |    R    |
+| batStateOfCharge          | Number:Dimensionless      | Current battery charge state  |    R    |
+| selfConsumption           | Number:Power              | Current self consumption  |    R    |
+
+The following Channels are writeable
+
+| Channel Type ID           | Item Type                 | Description                                                 |  Read Write  |
+|-------------------------------|---------------------------|-------------------------------------------------------|-----------------|    
+| chargeTimeEnd             | String          | Set charge time end, Value = 00:00  |    W    |
+| batteryType               | String          | Set battery type, Value = 1 = None, Value = 2 = PIKO Battery Li, Value = 3 = BYD B-Box HV  |   W    |
+| batteryUsageConsumption   | String          | Set battery usage consumption, Value = 100  |    W    |
```

Comment:
> It's not percent, it's text which can be sent to the inverter for configuration.

As the readme states "Value = 100" I guess this would be a numeric value. Can you explain in short what the feature behind the parameter is?